src/utils.rs

```diff
@@ -10,3 +11,97 @@ pub fn read_file<P: AsRef<Path>>(file_name: P) -> Result<String> {
     file.read_to_string(&mut data)?;
     Ok(data)
 }
+
+#[derive(Debug)]
+pub struct CommandOutput {
+    pub stdout: String,
+    pub stderr: String,
+}
+
+impl PartialEq for CommandOutput {
+    fn eq(&self, other: &Self) -> bool {
+        self.stdout == other.stdout && self.stderr == other.stderr
+    }
+}
+
+/// Execute a command and return the output on stdout and stderr if sucessful
+pub fn exec_cmd(cmd: &str, args: &[&str]) -> Option<CommandOutput> {
+    log::debug!("Executing command '{:?}' with args '{:?}'", cmd, args);
+    match Command::new(cmd).args(args).output() {
+        Ok(output) => {
+            let stdout_string = String::from_utf8(output.stdout).unwrap();
+            let stderr_string = String::from_utf8(output.stderr).unwrap();
+
+            if !output.status.success() {
+                log::debug!("Non-zero exit code '{:?}'", output.status.code());
```

Comment:
Likewise with these log statements.
I think they might add up to being a little too much for debug logs.